github-workflow-example.yaml

```diff
@@ -0,0 +1,14 @@
+name: Mergify Stack Autorebase
+on:
+  issue_comment:
+    types: [created, edited, deleted]
+
+jobs:
+  mergify-stack-auto-rebase:
+    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '@mergify-cli stack rebase')
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Run mergify stack github-action-auto-rebase
+        uses: Mergifyio/mergify-cli@main
+        with:
+          GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
does this need setup?

Reply:
I didn't get the question.

Reply:
Is `secrets.PERSONAL_ACCESS_TOKEN` always available? I don't get how you configure this 🤷

Reply:
It's a GitHub `secrets`, you need to set it in your GitHub organization or repository secrets.

Reply:
Ok, so there will be a single user doing all the rebases?
This really needs a proper documentation somewhere.

Reply:
I'm thinking about moving the action/workflow files into another repository, that will ease the documentation